README.md

```diff
@@ -1,6 +1,7 @@
 # Maven Plugins for Azure Services
 [![Travis CI](https://travis-ci.org/Microsoft/azure-maven-plugins.svg?branch=master)](https://travis-ci.org/Microsoft/azure-maven-plugins/) 
-[![AppVeyor](https://ci.appveyor.com/api/projects/status/qfpxt9gct33dfmns/branch/master?svg=true)](https://ci.appveyor.com/project/xscript/azure-maven-plugins)
+[![AppVeyor Function Plugin](https://ci.appveyor.com/api/projects/status/5jti4qwh0j4ekh72/branch/develop?svg=true)](https://ci.appveyor.com/project/xscript/azure-maven-plugins-vvy0i)

```

Comment:
I can not get what's the meaning of each badge. Please put them separately into a table.

You can take this as an example: https://github.com/Azure/azure-functions-core-tools